src/content/events/2023-12-08.mdx

```diff
@@ -0,0 +1,43 @@
+---
+date: 2023-12-08
+cover: '/2023-12-08.png'
+mdxMeta:
+  title: 'Eisbach Callin presents Posij'
+  description: 'Sunny Red, Feierwerk, Hansastraße 39, Munich'
+---
+
+## Time
+
+<RenderDoors timeDoors='22' price='18' />
+
+## Location
+
+<RenderLocation
+  venueName='Sunny Red'
+  address='Feierwerk Hansastr. 39'
+  city='Munich'
+  mapUrl='https://goo.gl/maps/mA8iv2XEHf7XfRLv7'
+/>
+
+## Lineup
+
+<RenderArtists
+  artists={[
+    {
+      name: 'POSIJ',
+      label: 'Vision',
+      href: 'https://soundcloud.com/posij',
+    },
+    { name: 'Miso' },
```

Comment:
```suggestion
    { name: 'Kamikatze b2b Polaster', label: 'Time Tripping' },
    { name: 'Miso' },
```

Reply:
i added them both individually, I guess this is okay - but there should exist a way to also express this on the web page